### DISCUSSION TITLE
Idea: Make our `apisix project` out of the box more reachable to users smoothly

### DISCUSSION THREAD

**Discussion by afzal442** [apache/apisix#9387] [Ideas]
As a user, I want to try `apisix API gateway` for my microservice based application on built-in console fully integrated with API gateway and other stuff installed.

Proposal:
We can create small exercise labs for different scenario based on apisix API gateway, ingress, etc. For that we can use [killercoda](https://killercoda.com/) env.

For reference: https://education.konghq.com/dashboard
![image](https://user-images.githubusercontent.com/11625672/234970758-4e40aa63-ce88-4031-9cb2-01e8846dbb9e.png)

cc @tao12345666333

**Comment by afzal442** [comment]
@everyone let us know if anyone is interested so we can work together to make it worthy. 💡

**Comment by tao12345666333** [comment]
SGTM.

**Reply by afzal442** [reply]
Thank you! 😄
